internal/cloudapi/openapi.yml

```diff
@@ -60,6 +60,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ComposeResult'
+  /packages:
+    post:
+      summary: Get packages
+      description: Get a list of packages for a given distribution and repositories
```

Comment:
Instead of getting the full list, should we make this a search, and limit the number of replies (possibly as an argument)? Returning 50000 packages (on fedora) might be a bit sluggish.